### ISSUE TITLE
Accordion Panel content is not accessible by screen reader once Header is expanded

### ISSUE THREAD

**Issue by RichieRunner** [nico3333fr/jquery-accessible-accordion-aria#37]
My screen reader is NVDA. I can expand the Header with `Enter` keypress, but how can I get my screen reader to access the content within the panel? It _should_ be accessible by `Down` keypress, but looks like it's not.

I tested with the accordions here.
